integration-tests/src/test/java/gov/uk/address/api/stepDefinitions/AddressApiHappyPath.java

```diff
@@ -1,408 +0,0 @@
-package gov.uk.address.api.stepDefinitions;
```

Comment:
Moving folders, but git can't figure that out :broken_heart:

Reply:
This file is deleted as part of the PR.